### PR TITLE
Молоткова С.С. 381806-1 ЛР №1

### PR DESCRIPTION
**Pull request by SMolotkova**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [x] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [x] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

File: modules/complex-number/test/molotkova_svetlana_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    EXPECT_EQ(rres.getIm(), eres.getIm());
+}
+TEST(Molotkova_Svetlana_ComplexNumberTesting, Multiply_by_number) {
+    ComplexNumber a(2.0, -7.0);
+    ComplexNumber b1(3.0, 0);
+    double b2 = 3.0;
+    ComplexNumber rres = a*b1;
+    ComplexNumber eres(a.getRe()*b2,a.getIm()*3);
+    bool res = rres == eres;
+    EXPECT_TRUE(res);
```
Comment (dkurt) [inline modules/complex-number/test/molotkova_svetlana_complex_number.cpp:22]: Use EXPECT_EQ please for comparison

File: modules/complex-number/test/molotkova_svetlana_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ComplexNumber eres(-5.0,12.0);
+    EXPECT_EQ(rres.getRe(), eres.getRe());
+    EXPECT_EQ(rres.getIm(), eres.getIm());
+}
+TEST(Molotkova_Svetlana_ComplexNumberTesting, Multiply_by_number) {
+    ComplexNumber a(2.0, -7.0);
+    ComplexNumber b1(3.0, 0);
+    double b2 = 3.0;
+    ComplexNumber rres = a*b1;
+    ComplexNumber eres(a.getRe()*b2,a.getIm()*3);
```
Comment (dkurt) [inline modules/complex-number/test/molotkova_svetlana_complex_number.cpp]: Why `b2` variable is used only for real part?

File: modules/complex-number/test/molotkova_svetlana_complex_number.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    ComplexNumber rres = a*b1;
+    ComplexNumber eres(a.getRe()*b2, a.getIm()*3);
+    bool res = rres == eres;
+    EXPECT_TRUE(res);
+}
+
+TEST(Molotkova_Svetlana_ComplexNumberTesting, No_Exeption) {
+    ComplexNumber a1(2.0, 9.0);
+    ComplexNumber a2(3.0, 3.0);
+    ASSERT_NO_THROW(a1 / a2);
```
Comment (dkurt) [inline modules/complex-number/test/molotkova_svetlana_complex_number.cpp:28]: Please do sanity check for the result as well.

File: modules/complex-number/test/molotkova_svetlana_complex_number.cpp (hunk 2 above)
Comment (SMolotkova) [reply modules/complex-number/test/molotkova_svetlana_complex_number.cpp]: fix

File: modules/complex-number/test/molotkova_svetlana_complex_number.cpp (hunk 1 above)
Comment (SMolotkova) [reply modules/complex-number/test/molotkova_svetlana_complex_number.cpp]: fix

File: modules/complex-number/test/molotkova_svetlana_complex_number.cpp (hunk 3 above)
Comment (SMolotkova) [reply modules/complex-number/test/molotkova_svetlana_complex_number.cpp:28]: fix

**Review by BelikJulia — APPROVED** [review]
Все хорошо

**Review by ParanichevaAlyona — APPROVED** [review]
Видимо теперь все найс

**Pull request merged**